versioned/storage/cache/src/main/java/org/projectnessie/versioned/storage/cache/CaffeineCacheBackend.java

```diff
@@ -110,18 +127,35 @@ public long expireAfterRead(
                   singletonList(Tag.of("cache", CACHE_NAME)),
                   "",
                   x -> config.capacityMb());
+              meterRegistry.gauge(
+                  "current_weight",
```

Comment:
Could you make this `current_weight_mb` to be comparable to `cache_capacity_mb`?

Also, how about renaming to `cache_weight_mb`?.. "current" is only at the time of measurement :)

Reply:
Can rename to _mb - but i don't think that renaming an existing metric's a good idea ;)